bazelrio/libraries/cpp/ctre/phoenix/BUILD

```diff
@@ -8,7 +8,7 @@ cc_library(
         "@__bazelrio_com_ctre_phoenix_cci_headers//:headers",
         "@__bazelrio_com_ctre_phoenix_core_headers//:headers",
         "@__bazelrio_com_ctre_phoenix_wpiapi-cpp_headers//:headers",
-        "@bazelrio//libraries/cpp/wpilib/wpilibc",
+        "//libraries/cpp/wpilib/wpilibc",
```

Comment:
I've never really looked into how this works, if people just happen to get lucky or not. If I have a `libraries` directory in my project, bazel is smart enough to know that this line corresponds to its actual workspace instead of the `__main__` workspace?

Reply:
I think this is safe. `http_archive` populates `bazel-<project>/external/` with directories containing an empty WORKSPACE files. Any BUILD files under those trees will resolve `//` to their specific workspace root.